Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -919,9 +919,8 @@ open class CarPlayNavigationViewController: UIViewController, BuildingHighlighti
         
         // Add maneuver arrow
         if #available(iOS 13.0, *) {
-            let userInterfaceStyle = traitCollection.userInterfaceStyle
```

Comment:
I'm just wondering if we have any CarPlay screenshot tests.

Reply:
We have maneuver view snapshot test, but for CarPlay it is presented not like a view, but through the generated image, and that's why it was not updated as the rest of the views. Using image instead of a view is a CarPlay specifics